pkg/test/framework/components/cluster/clusters.go

```diff
@@ -183,3 +185,12 @@ func (c Clusters) filterClusters(included func(Cluster) bool,
 func (c Clusters) String() string {
 	return fmt.Sprintf("%v", c.Names())
 }
+
+// The function validates if the cluster is a "Kind" cluster,
+// By looking into a context name. Expects "kind-" prefix.
+// That is required by some tests for specific actions on "Kind".
+func (c Clusters) IsKindCluster() bool {
+	config, _ := clientcmd.NewDefaultClientConfigLoadingRules().Load()
```

Comment:
Hmm.. this is still just reading from the users kubeconfig? Moving the function doesn't change anything.

Istio's test framework does not use the standard config loading rules, so anything using that is broken.

Reply:
Might be slightly better though so if its good enough for all existing use cases then I suppose its fine.